khard/data/template.yaml

```diff
@@ -2,6 +2,10 @@
 # from the full name below if not given.
 Formatted name : 
 
+# kind (requires vcard >= 4.0)
+# one of: individual, group, org, location, application, device
```

Comment:
where did you get "application" and "device" from? I can not see them in the rfc
```suggestion
# one of: individual, group, org, location
```

Reply:
I would reference the rfc in that comment and also at the top of the file.